packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -137,11 +138,13 @@ interface PendingBlob {
 	status: PendingBlobStatus;
 	storageId?: string;
 	handleP: Deferred<BlobHandle>;
-	uploadP?: Promise<ICreateBlobResponse>;
+	uploadP?: Promise<ICreateBlobResponse | void>;
 	uploadTime?: number;
 	minTTLInSeconds?: number;
 	attached?: boolean;
 	acked?: boolean;
+	abortSignal?: AbortSignal;
+	opsent?: boolean;
```

Comment:
not necessary for this PR, but in the next PR it would be good to clean up some of the properties here. maybe we could unify status, acked, opsent, and attached all on status. we might need to keep a distinct state for attached, since network state, and attach state can be matrix in the offline case.

Reply:
I'll create an ADO item for that